### PR TITLE
feat(claude): map Claude Code model strings to OpenCode format when importing agents

### PR DESCRIPTION
**Pull request by devxoul**
## Summary

- When importing Claude Code agents (from `.claude/agents/` or plugins), the `model` frontmatter field was parsed but silently dropped
- Added `mapClaudeModelToOpenCode()` that converts Claude Code model strings to OpenCode's `provider/model` format
- Handles all Claude Code model conventions per [official docs](https://docs.anthropic.com/en/docs/claude-code/sub-agents#choose-a-model):
  - **Aliases**: `sonnet` → `anthropic/claude-sonnet-4-6`, `opus` → `anthropic/claude-opus-4-6`, `haiku` → `anthropic/claude-haiku-4-5` (case-insensitive)
  - **`inherit`**: returns `undefined` (no model override, lets OpenCode decide)
  - **Bare Claude model IDs**: `claude-sonnet-4-5-20250514` → `anthropic/claude-sonnet-4-5-20250514` (adds provider prefix)
  - **Dot normalization**: `claude-3.5-sonnet` → `anthropic/claude-3-5-sonnet`
  - **Already-prefixed**: `anthropic/claude-sonnet-4-6` passes through unchanged
  - **Non-Claude models**: dot normalization only, no prefix guessing
- Integrated into both `claude-code-agent-loader` and `claude-code-plugin-loader` agent loaders
- 19 tests covering aliases, inherit, bare IDs, dot normalization, passthrough, and edge cases

### CODE REVIEW

File: src/features/claude-code-agent-loader/claude-model-mapper.test.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      expect(mapClaudeModelToOpenCode("claude-opus-4-6")).toBe("claude-opus-4-6")
+    })
+
+    it("#when called with claude-haiku-4-5 #then returns unchanged", () => {
+      expect(mapClaudeModelToOpenCode("claude-haiku-4-5")).toBe("claude-haiku-4-5")
+    })
+  })
+
+  describe("#given non-Claude model", () => {
+    it("#when called with gpt-5.2 #then normalizes dots only", () => {
```
[the remark this answers is not in the document]
Comment (devxoul) [reply src/features/claude-code-agent-loader/claude-model-mapper.test.ts]: This is only for claude models

File: src/features/claude-code-plugin-loader/agent-loader.ts [hunk 2]
```diff
@@ -46,10 +47,13 @@ export function loadPluginAgents(plugins: LoadedPlugin[]): Record<string, AgentC
         const originalDescription = data.description || ""
         const formattedDescription = `(plugin: ${plugin.name}) ${originalDescription}`
 
+        const mappedModel = mapClaudeModelToOpenCode(data.model)
+
         const config: AgentConfig = {
           description: formattedDescription,
           mode: "subagent",
           prompt: body.trim(),
+          ...(mappedModel && { model: mappedModel }),
```
[the remark this answers is not in the document]
Comment (devxoul) [reply src/features/claude-code-plugin-loader/agent-loader.ts]: This is not for other opencode models
